src/control/Control.cpp

```diff
@@ -424,6 +425,13 @@ void Control::actionPerformed(ActionType type, ActionGroup group, GdkEvent* even
             showSettings();
             break;
 
+        case ACTION_ARRANGE_BRING_TO_FRONT:
+        case ACTION_ARRANGE_BRING_FORWARD:
+        case ACTION_ARRANGE_SEND_BACKWARD:
+        case ACTION_ARRANGE_SEND_TO_BACK:
+            this->reorderSelection(type);
+            break;
```

Comment:
This won't generate any warnings because there is no code being executed through the fallthrough cases (in any case, it is quite clear that the fallthrough is occurring as intended). Consider the following sample program:
```c++
int main() {
    int a = 0;
    int b = 0;
    switch (a) {
        case 0:
        case 1:
        case 2:
            b = 1;
            break;
        default:
            b = 2;
            break;
    }
    return b;
}
```
If you compile with `clang++ -Wimplicit-fallthrough example.cpp` (or `g++`), there will be no warning. If you insert `b = 3` after `case 0`, then there will be a warning.